src/test/java/tictactoe_game/WinnerTest.java

```diff
@@ -0,0 +1,201 @@
+package tictactoe_game;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static tictactoe_game.model.Element.*;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import tictactoe_game.implementation.WinnerImplementation;
+import tictactoe_game.model.Element;
+import tictactoe_game.model.Grid;
+import tictactoe_game.model.types.*;
+
+public class WinnerTest {
```

Comment:
I am not sure that this is the correct name for the class, since we are testing the _game over_ functionality, and not deciding who won.

Please, rename it.

Reply:
Please, ignore the comment above. The name _is_ correct.